solana/gateway-ts/src/lib/util.ts

```diff
@@ -51,6 +52,7 @@ export const defaultRetryConfig = {
 };
 
 export const runFunctionWithRetry = async (
+  methodName: string, // for logging purposes so we know which call this was
```

Comment:
Just a suggestion and non blocking. Instead of having to pass in a method name with each call to runFunctionWithRetry can we not log before the function is called wherever it is called, we can just use the catch to log. i.e

```
// So instead of
return runFunctionWithRetry(
  "sendTransaction",
  fn,
  SOLANA_COMMITMENT,
  customRetryConfig
)

// We have 
...
console.log(`Trying Solana connection method '${methodName}'`);

return runFunctionWithRetry(fn, SOLANA_COMMITMENT, customRetryConfig)
    .catch((e) => {
        console.error(
           `Retries exhausted in Solana connection method '${methodName}' .`,
           { error: e }
         );
    });
```

Reply:
That's true, but I also want it to include the method name with each "attempt x of y" call which happens inside the retry fn.